versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx

```diff
@@ -15,6 +15,23 @@ To achieve the goal we need to ensure that:
 - Old code is compatible with new database schema or new database schema is compatible with old code.
 - Migrations don't lock tables, rows etc. for no longer than one second.
 
+## Task Autodiscovery in Zero Downtime Migrations
```

Comment:
Maybe it would be better to merge that whole section with the `Data migration` paragraph? WDYT?

Reply:
done